hack/test/e2e-aws.sh

```diff
@@ -78,7 +78,7 @@ function setup {
 
   ${CLUSTERCTL} config cluster ${NAME_PREFIX} \
     --kubeconfig /tmp/e2e/docker/kubeconfig \
-    --from https://github.com/talos-systems/cluster-api-templates/blob/v1alpha4/aws/standard/standard.yaml > ${TMP}/cluster.yaml
+    --from https://github.com/rsmitty/cluster-api-templates/blob/main/aws/standard/standard.yaml > ${TMP}/cluster.yaml
```

Comment:
will amend once I run everything through testing and get templates merged upstream.